scene/gui/base_button.cpp

```diff
@@ -216,9 +216,7 @@ void BaseButton::set_pressed(bool p_pressed) {
 	if (p_pressed) {
 		_unpress_group();
 	}
-	if (toggle_mode) {
```

Comment:
We already have an early return for `!toggle_mode` above, so this appeared redundant.